WordPressShared/WordPressShared/Core/Views/WPStyleGuide+DynamicType.swift

```diff
@@ -75,6 +75,28 @@ extension WPStyleGuide {
         return UIFont(descriptor: fontDescriptor, size: CGFloat(0.0))
     }
 
+    /// Creates a UIFont for the user current text size settings and a maximum font size
+    ///
+    /// - Parameters:
+    ///     - style: The desired UIFontTextStyle.
+    ///     - maximumPointSize: The biggest font size allowed.
+    ///
+    /// - Returns: The created font.
+    ///
+    @objc public class func fontForTextStyle(_ style: UIFontTextStyle, maximumPointSize: CGFloat) -> UIFont {
```

Comment:
Can we turn this into an improvement to `fontForTextStyle(_: fontWeight:)` or `fontForTextStyle(_: symbolicTraits:)` with a new parameter with a default value, as opposed to adding a new method? Especially since we want to add better dymaic text support app-wide, seems like these existing methods should learn new tricks?

Reply:
Definitely. I see the point: not adding more methods to the API, and at the same time, getting the behaviour (limiting the font size) for free in the existing ones. I was so afraid of breaking all the things, that I didn't dare to do it. 😊